pages/common/convert.md

```diff
@@ -16,9 +16,9 @@
 
 `convert {{path/to/input_image.png}} -resize 640x480 {{path/to/output_image.png}}`
 
-- Horizontally append images:
+- Scalte an image to have a certain filesize:
 
-`convert {{path/to/image1.png path/to/image2.png ...}} +append {{path/to/output_image.png}}`
+`convert {{path/to/input_image.png}} -define jpeg:extent=512kb {{path/to/output_image.jpg}}`
 
 - Vertically append images:
```

Comment:
I think if we keep both horizontal and vertical append examples we will have more than 6 examples, so I chose to remove that one, because its similar to horizontal append. But we can also make some other choice.

Reply:
Ah that makes sense